### ISSUE TITLE
icon-version plugin is not working with latest android gradle plugin

### ISSUE THREAD

**Issue by lhoracek** [akonior/icon-version#10]
After creating new project with ` classpath 'com.android.tools.build:gradle:3.0.0-alpha4'` the project won't build with icon-version plugin applied

```
* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Could not get unknown property 'resDir' for task ':app:processDebugResources' of type com.android.build.gradle.tasks.ProcessAndroidResources.

```

**Comment by andreashermann** [comment]
With the beta version ```com.android.tools.build:gradle:3.0.0-beta2``` an additional error occurs before even accessing the property **resDir** when running the icon-version plugin. 

```
Execution failed for task ':app:process*****TestDebugResources'.
> Could not get unknown property 'manifestOutputFile' for task ':app:process*****TestDebugManifest' of type com.android.build.gradle.tasks.MergeManifests.
```

```
Caused by: groovy.lang.MissingPropertyException: Could not get unknown property 'manifestOutputFile' for task ':app:process*****TestDebugManifest' of type com.android.build.gradle.tasks.MergeManifests.
        at org.gradle.internal.metaobject.AbstractDynamicObject.getMissingProperty(AbstractDynamicObject.java:83)
        at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:61)
        at com.android.build.gradle.tasks.MergeManifests_Decorated.getProperty(Unknown Source)
        at pl.itako.iconversion.IconVersionPlugin$_apply_closure1_closure2_closure3_closure4.doCall(IconVersionPlugin.groovy:49)
        at pl.itako.iconversion.IconVersionPlugin$_apply_closure1_closure2_closure3.doCall(IconVersionPlugin.groovy:48)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:681)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:656)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:122)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:197)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:107)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:111)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
        ... 27 more
```

It seems API changes in the Android Gradle Plugin are responsible for the incompatibility.
https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#variant_api
> The processManifest.manifestOutputFile() method is no longer available, and you get the following error when you try to call it:

**Comment by n-belokopytov** [comment]
Same problem here

**Comment by Tooto** [comment]
same

**Comment by ypaiag** [comment]
same too(((

**Comment by amal** [comment]
With Gradle plugin v3.0.0
```
java.lang.RuntimeException: Manifest Tasks does not support the manifestOutputFile property any more, please use the manifestOutputDirectory instead.
For more information, please check https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html
	at com.android.build.gradle.tasks.ManifestProcessorTask.getManifestOutputFile(ManifestProcessorTask.java:128)
	at com.android.build.gradle.tasks.MergeManifests_Decorated.getManifestOutputFile(Unknown Source)
	at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.getProperty(BeanDynamicObject.java:228)
	at org.gradle.internal.metaobject.BeanDynamicObject.tryGetProperty(BeanDynamicObject.java:171)
	at org.gradle.internal.metaobject.CompositeDynamicObject.tryGetProperty(CompositeDynamicObject.java:55)
	at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:59)
	at com.android.build.gradle.tasks.MergeManifests_Decorated.getProperty(Unknown Source)
	at pl.itako.iconversion.IconVersionPlugin$_apply_closure1_closure2_closure3_closure4.doCall(IconVersionPlugin.groovy:49)
	at pl.itako.iconversion.IconVersionPlugin$_apply_closure1_closure2_closure3.doCall(IconVersionPlugin.groovy:48)
	at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:722)
	at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:695)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:121)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
	at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:110)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
	... 29 more
```

**Comment by Loyea** [comment]
@amal  same problem.

**Comment by ewoks** [comment]
Just a poke to maintainer so this high priority ticket is not forgotten

**Comment by akonior** [comment]
Fixed in version `1.1.5`
